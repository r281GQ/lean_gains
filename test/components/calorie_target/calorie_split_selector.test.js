import React from 'react';
import { shallow } from 'enzyme';
import CalorieSplitSelector from './../../../src/components/calorie_target/calorie_split_selector';

describe('CalorieSplitSelector test', () => {
  it('should render', () => {
    const component = shallow(
      <CalorieSplitSelector/>
    );
    expect(component).toHaveLength(1);
    expect(component).toMatchSnapshot();
  });
});
