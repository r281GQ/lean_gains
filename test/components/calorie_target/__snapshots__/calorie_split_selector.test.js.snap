// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalorieSplitSelector test should render 1`] = `
<Field
  component={[Function]}
  floatingLabelText="Select your macro target"
  fullWidth={true}
  name="calorieSplit"
>
  <MenuItem
    anchorOrigin={
      Object {
        "horizontal": "right",
        "vertical": "top",
      }
    }
    checked={false}
    desktop={false}
    disabled={false}
    focusState="none"
    insetChildren={false}
    primaryText="Recomposition (-20%/+20%)"
    targetOrigin={
      Object {
        "horizontal": "left",
        "vertical": "top",
      }
    }
    value="recomp"
  />
  <MenuItem
    anchorOrigin={
      Object {
        "horizontal": "right",
        "vertical": "top",
      }
    }
    checked={false}
    desktop={false}
    disabled={false}
    focusState="none"
    insetChildren={false}
    primaryText="Cut (-30%/10%)"
    targetOrigin={
      Object {
        "horizontal": "left",
        "vertical": "top",
      }
    }
    value="cut"
  />
  <MenuItem
    anchorOrigin={
      Object {
        "horizontal": "right",
        "vertical": "top",
      }
    }
    checked={false}
    desktop={false}
    disabled={false}
    focusState="none"
    insetChildren={false}
    primaryText="Slow bulk (-10%/+30%)"
    targetOrigin={
      Object {
        "horizontal": "left",
        "vertical": "top",
      }
    }
    value="slowbulk"
  />
  <MenuItem
    anchorOrigin={
      Object {
        "horizontal": "right",
        "vertical": "top",
      }
    }
    checked={false}
    desktop={false}
    disabled={false}
    focusState="none"
    insetChildren={false}
    primaryText="Custom values"
    targetOrigin={
      Object {
        "horizontal": "left",
        "vertical": "top",
      }
    }
    value="custom"
  />
</Field>
`;
